spec/models/case/sar/offender_spec.rb

```diff
@@ -72,6 +72,15 @@
     end
   end
 
+  describe "#set_number" do
+    it "assigns a case number to rejected offender sar" do
```

Comment:
This test does not test the set_number method. You can verify this by making a change to the method - e.g. change the prefix letter from "R" to something else - and seeing that this test still passes.